From 053d333cfbdf4212720355a214cc6c57c2d380bf Mon Sep 17 00:00:00 2001
From: Pritesh Patel <pritesh.patel@einfochips.com>
Date: Wed, 4 Sep 2024 08:28:24 +0000
Subject: [PATCH 043/128] defconfig: hifive-premier-p550: Configure autofs4
 driver as inbuilt

When autofs04 driver is built as module, it does not load
automatically, resulting in following systemd error:

[4.535869] systemd[1]: Failed to find module 'autofs4'

Hence, configuring it as inbuilt

Signed-off-by: Pritesh Patel <pritesh.patel@einfochips.com>
---
 arch/riscv/configs/hifive-premier-p550_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/configs/hifive-premier-p550_defconfig b/arch/riscv/configs/hifive-premier-p550_defconfig
index 087d3a93fb3a..a7db5c12908f 100644
--- a/arch/riscv/configs/hifive-premier-p550_defconfig
+++ b/arch/riscv/configs/hifive-premier-p550_defconfig
@@ -779,7 +779,7 @@ CONFIG_FANOTIFY_ACCESS_PERMISSIONS=y
 CONFIG_QUOTA_NETLINK_INTERFACE=y
 CONFIG_QFMT_V1=m
 CONFIG_QFMT_V2=m
-CONFIG_AUTOFS_FS=m
+CONFIG_AUTOFS_FS=y
 CONFIG_FUSE_FS=m
 CONFIG_CUSE=m
 CONFIG_VIRTIO_FS=m
-- 
2.47.0

